mne/preprocessing/tests/test_maxwell.py

```diff
@@ -49,7 +50,7 @@ def test_maxwell_filter():
                          allow_maxshield=True)
         raw_err = Raw(raw_fname, preload=False, proj=True,
                       allow_maxshield=True).crop(0., 0.1, False)
-    assert_raises(RuntimeError, maxwell.maxwell_filter, raw_err)
```

Comment:
we don't typically do this in tests, so I imported everything necessary manually